swiftnav/tests/reference_frames.rs

```diff
@@ -142,13 +142,363 @@ fn euref_etrf2014() {
     );
     assert_float_eq!(
         result_coords.velocity().as_ref().unwrap().z(),
-        0.0200,
+        0.0198,
```

Comment:
Did this value change because of a bug in a prior implementation or are you just bringing the threshold closer to the expected?

Reply:
The transformation being performed has changed, it was ITRF2020 to ETRF2014 but is now ITRF**_2014_** to ETRF2014 so the result is slightly different.